library/src/main/java/com/schibsted/spain/barista/assertion/BaristaRecyclerViewAssertions.kt

```diff
@@ -1,15 +1,92 @@
 package com.schibsted.spain.barista.assertion
 
+import android.content.res.Resources
 import android.support.annotation.IdRes
 import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.support.v7.widget.RecyclerView
+import android.view.View
 import com.schibsted.spain.barista.internal.matcher.DisplayedMatchers.displayedWithId
 import com.schibsted.spain.barista.internal.matcher.RecyclerViewItemCountAssertion
+import org.hamcrest.CoreMatchers
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
 
 object BaristaRecyclerViewAssertions {
 
-    @JvmStatic
-    fun assertRecyclerViewItemCount(@IdRes recyclerViewId: Int, expectedItemCount: Int) {
-        onView(displayedWithId(recyclerViewId)).check(RecyclerViewItemCountAssertion(expectedItemCount))
-    }
+  @JvmStatic
+  fun assertRecyclerViewItemCount(@IdRes recyclerViewId: Int, expectedItemCount: Int) {
+    onView(displayedWithId(recyclerViewId)).check(RecyclerViewItemCountAssertion(expectedItemCount))
+  }
+
+  @JvmStatic
+  fun assertDisplayedAtPosition(@IdRes recyclerViewId: Int, position: Int, text: String) {
+    onView(withId(recyclerViewId))
+        .perform(scrollToPosition<RecyclerView.ViewHolder>(position))
+
+    onView(atPosition(recyclerViewId, position))
+        .check(matches(CoreMatchers.anyOf(withChild(withText(text)), withText(text))))
+
+  }
+
+  @JvmStatic
+  fun assertDisplayedAtPosition(@IdRes recyclerViewId: Int, position: Int, viewId: Int, text: String) {
+    onView(withId(recyclerViewId))
+        .perform(scrollToPosition<RecyclerView.ViewHolder>(position))
+
+    onView(atPositionOnView(recyclerViewId, position, viewId))
+        .check(matches(withText(text)))
+  }
+
+  /**
+   * Fork of the RecyclerViewMatcher from https://gist.github.com/baconpat/8405a88d04bd1942eb5e430d33e4faa2
+   */
+  private fun atPosition(@IdRes recyclerViewId: Int, position: Int): Matcher<View> = atPositionOnView(recyclerViewId, position, -1)
```

Comment:
I'd suggest to use default params here so you'll have only one method `atPosition` with the default value of the `viewId = -1`. In addition I'd extract that `-1` to a constant named for example `NO_VIEW_ID`.
If you do that, you'll be able to do the same for `assertDisplayedAtPosition` and you'll simplify the code.